Packs/Core/ReleaseNotes/3_0_86.md

```diff
@@ -0,0 +1,36 @@
+
+#### Playbooks
+
+##### New: Uncommon remote scheduled task created
+
+- New: This playbook is designed to handle the alert "Uncommon remote scheduled task created".
+
+Playbook Stages:
+
+Analysis:
+
+- The playbook checks if the remote IP is external or has a bad reputation.
+
+Investigation:
+During the alert investigation, the playbook will perform the following:
+
+- Searches for related XDR alerts to the endpoint using the following MITRE techniques to identify any malicious activity: T1202 - Indirect Command Execution, T1021 - Remote Services.
+- Searches for related XSIAM agent alerts on the remote endpoint to determine if the creation of the scheduled task is part of an attack pattern.
+- Searches for suspicious command-line parameters indicating a malicious scheduled task.
+
+Remediation:
+
+- Remediation actions will disable the scheduled task and block the malicious IP, and close the alert.
+
+Requirements:
+
+For response actions, the following integrations are required: 
+
+- PAN-OS.
+
+
+#### Triggers Recommendations
+
+##### New: Uncommon remote scheduled task created
+
+- New: This trigger is responsible for handling 'Uncommon remote scheduled task created.
```

Comment:
This trigger runs the **Uncommon remote scheduled task created** Playbook to handle these alerts.